### PR TITLE
fix support for datatype CHOICE 

### PR DESCRIPTION
**Pull request by johnkop**

### CODE REVIEW

File: blpapijs.cpp [hunk 1]
```diff
@@ -1509,6 +1509,12 @@ Session::elementValueToValue(Isolate *isolate,
         }
         case blpapi::DataType::SEQUENCE:
             return elementToValue(isolate, e.getValueAsElement(idx));
+		
+		case blpapi::DataType::CHOICE:{
```
Comment (yupinghu) [inline blpapijs.cpp]: It appears you are using tabs instead of spaces; please switch to spaces.

File: blpapijs.cpp [hunk 2]
```diff
@@ -1509,6 +1509,12 @@ Session::elementValueToValue(Isolate *isolate,
...
+			if (e.isArray()) {
+				return elementToValue(isolate, e.getValueAsElement(idx));
+			}
+		}
```
Comment (yupinghu) [inline blpapijs.cpp]: Please add a `break`.

**Comment by yupinghu** [comment]
Thanks for the contribution! I've added some comments re: code style and others may have comments as well.

**Comment by apaprocki** [comment]
Looks good now.. can you just `git commit --amend` and edit the commit message to be something logical? (After doing that you'll need to `push -f` to force it.) If we merge this right now it will just show up as "my commit" in the history.

**Comment by ericvw** [comment]
Closing this in favor of #66.

**Pull request closed without merging**